battleship-client/src/main/java/com/epam/solid/nie/client/communication/SocketServer.java

```diff
@@ -48,12 +55,30 @@ public Cell receiveFirstMove() {
         return cells.poll();
     }
 
+    @Override
+    public void sendGameOverToOpponent() {
+        server.sendGameOverToOpponent();
+    }
+
     private void receiveAllMovesWithoutSending() {
         allMoves = "";
         String moves = server.getEnemyShips();
+        if (moves.equals("Q")) {
```

Comment:
String equals and capital letter. Intentional? Or did you mean "q" as well?

Reply:
Intentionally. Is there any problem with it?

Reply:
Just making sure it's not a typical mistake of "equals vs equalsIgnoreCase".